test/verify/check-storage-stratis

```diff
@@ -337,6 +337,47 @@ class TestStorageStratis(StorageCase):
         # Check that the entry has disappeared from fstab
         self.assertEqual(m.execute("grep /run/fsys1 /etc/fstab || true"), "")
 
+    def testAlerts(self):
+        m = self.machine
+        b = self.browser
+
+        self.login_and_go("/storage")
+
+        dev_1 = "/dev/sda"
+        m.add_disk("4G", serial="DISK1")
+        b.wait_in_text("#drives", dev_1)
+
+        dev_2 = "/dev/sdb"
+        m.add_disk("4G", serial="DISK2")
+        b.wait_in_text("#drives", dev_2)
+
+        # Create an encrypted pool with two block devices
+        self.dialog_open_with_retry(trigger=lambda: self.devices_dropdown("Create Stratis pool"),
+                                    expect=lambda: (self.dialog_is_present('disks', dev_1) and
+                                                    self.dialog_check({"name": "pool0"})))
+        self.dialog_set_val("encrypt.on", True)
+        self.dialog_set_val("passphrase", "foodeeboodeebar")
+        self.dialog_set_val("passphrase2", "foodeeboodeebar")
+        self.dialog_set_val("disks", {dev_1: True})
+        self.dialog_set_val("disks", {dev_2: True})
+        b.assert_pixels("#dialog", "create-encrypted-pool")
+        self.dialog_apply()
+        self.dialog_wait_close()
+
+        b.wait_in_text("#devices", "pool0")
+        b.wait_not_present('#pool-alert')
+
+        m.execute('JSON=$(sudo cryptsetup token export --token-id=1 /dev/sda | jq \'.key_description = "stratis-1-key-no-other-is-the-same"\'); sudo cryptsetup token remove --token-id=1 /dev/sda; echo $JSON | sudo cryptsetup token import --token-id=1 /dev/sda; systemctl restart stratisd')
+
+        b.click('.sidepanel-row:contains("pool0")')
+        b.wait_visible('#pool-alert')
+        b.click("button:contains(Create new filesystem)")
+        self.dialog({'name': 'fsys1',
```

Comment:
Instead of adding parameters to `self.dialog`, the tests usually just write out the actions explicitly with `self.dialog_wait_open`, `self.dialog_set_val`, etc, like you are doing above when creating the pool. I think we should continue to do that.